tools/deps/prebuilt/include/dispatch/continuation_cache.h

```diff
@@ -4,38 +4,44 @@
 // If dc_vtable is less than 127, then the object is a continuation.
 // Otherwise, the object has a private layout and memory management rules. The
 // first two words must align with normal objects.
-#define DISPATCH_CONTINUATION_HEADER(x) \
-    const void* do_vtable;              \
-    struct x* volatile do_next;         \
-    dispatch_function_t dc_func;        \
-    void* dc_ctxt
+#define DISPATCH_CONTINUATION_HEADER(x)	\
```

Comment:
this file still got reformatted.

Reply:
Believe it or not, the file was formatted incorrectly before. The file in deps\3rdparty\libdispatch\ is formatted to this. My assumption is that this file should be the most accurate since the one checked into tools\deps\prebuilt\include must be a copy. Copying over this file is why it changed and will happen each time libdispatch is built. This formatting most closely resembles the original fork of dr pizza's libdispatch repo.

Should I apply the (clearly better looking) formatting that was applied to the copy of this header to the original header?

Reply:
As is the same for each "formatted" file in the tools folder.

Reply:
Ah, I see. Then this is a change we have to take.

Make sure it’s part of the “update prebuilts” commit :)